src/app/page.tsx

```diff
@@ -183,6 +183,64 @@ export default function Page() {
             })}
           </div>
         </Section>
+        <Section>
```

Comment:
whole section should be conditionally rendered to cover the case if there are no certifications

Reply:
Added conditional Data render for the Certifications and Publications section.